frontend/js/src/common/brainzplayer/SoundcloudPlayer.tsx

```diff
@@ -264,6 +268,7 @@ export default class SoundcloudPlayer
     callbackFunction: () => void
   ): Promise<void> => {
     const { refreshSoundcloudToken, onTrackNotFound, handleError } = this.props;
```

Comment:
If you feel so inclined, since you are accessing the global context here you could also use the APIService directly, and get rid of this `refreshSoundcloudToken` prop which is just `APIService.refreshSoundcloudToken`.
And the same goes for the other services.
We previously made it this way with the idea of possibly deploying brainzplayer as a separate JS package, hence no relying on global context.
However it's been a few years and there has been no need for a separate package, so let's make it cleaner...

Anyway, could be in this PR or later if you're busy with other stuff, it's nothing crucial.

Reply:
Yes, we can get rid of the props, but I've already made these changes in #2827 PR, so it does not make sense to make these changes here.